app/components/widgets/settings/bool_setting.js

```diff
@@ -104,21 +104,23 @@ export default class BoolSetting extends PureComponent {
         }
 
         return (
-            <View style={padding(isLandscape)}>
-                {labelContent}
+            <View>
```

Comment:
This View is extra you can change it to a <React.Fragment>

Reply:
Good catch, thank you!